lib/execution_engine2/db/models/models.py

```diff
@@ -336,7 +336,11 @@ class Job(Document):
         default=False
     )  # Marked true when all retry steps have completed
 
-    meta = {"collection": "ee2_jobs"}
+    # See https://docs.mongoengine.org/guide/defining-documents.html#indexes
+    # Hmm, are these indexes need to be + or - ?
```

Comment:
Depends on if you want sorted results - if not, it doesn't really matter. With the status/queued index, if you do an equality query on status and a range query on queued you can sort descending without having to do an in-memory sort.

Reply:
Ok, with the change now oldest first sorts can be done without an in-memory sort